src/components/HomePage.js

```diff
@@ -2,35 +2,90 @@ import styled from 'styled-components';
+import { CardGroup, Card} from 'react-bootstrap';
+
+
+
+import "./style.css";
 
-// For now, very basic home page with a button that leads to the module selection page
 const HomePage = () => {
     // useNavigate hook will be used to programmatically route the user based on their actions
     const navigate = useNavigate();
     return (
-        <Wrap>
-            <HeadingWrap>
-                Home Page
-            </HeadingWrap>
-            <Button
-                as={Col}
-                variant="primary"
-                onClick={() => {
-                    // When this button is clicked, route the user to the module selection page
-                    navigate('/modules');
-                }}
-            >
-                Module Selection Page
-            </Button>
+        
+        <Wrap>     
+            <Heading>
+               <strong>&nbsp;&nbsp;&nbsp;&nbsp;NeoCirc</strong>
+            </Heading>
+            <SubHeading2 width="95%">
+            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+            <i>An educational platform leveraging e-learning to promote proficiency in medical procedures.</i>
+            </SubHeading2>
+            <div style={{
+                alignItems: 'center',
+                justifyContent: 'center',
+                padding: '105px'
+            }}>
+            <CardGroup>
+                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+            <Card>
+                <Card.Body>
+                <center><Card.Title>The NeoCirc Experience</Card.Title></center>
+                <Card.Text>
+                    <br></br>
+                    <center>We strive to offer the most intimate and personalized experience possible, to do so we need to know more about you.</center>
+                    <br></br>
+                    <center><i>Which best describes you?</i></center>
+                    <br></br>
+                    <center>
+                    <Button as={Col}
+                        variant="primary"
+                        onClick={() => {
+                        // When this button is clicked, route the user to the module selection page
+                        navigate('/modules');
+                    }}
+                    >I am a Parent</Button>{' '}
+                    <Button as={Col}
+                    variant="dark"
+                    onClick={() => {
+                        // When this button is clicked, route the user to the module selection page
+                        navigate('/modules');
+                    }}
+                    >I am a Doctor</Button>
+                    </center>
+
+                </Card.Text>
+                </Card.Body>
+                
+            </Card>
+            </CardGroup>
+            </div>
+            
+    
+            
         </Wrap>
     );
 };
 
 const Wrap = styled.div`
+    
```

Comment:
```suggestion
    padding-left: 20px;
```
or whatever padding left necessary to account for the `&nbsp`